.gitignore

```diff
@@ -263,15 +263,30 @@ make_deb
 tmp/
 dist/
 *~
+linux/ibus-kmfl/m4/codeset.m4
```

Comment:
Can you move all the ignores pertaining to Linux into
`linux/.gitignore`? Other than Windows, we've kept each platform in their own .gitignore file.

Reply:
Very good catch -- in fact the earlier lines may break other platforms right now.

Reply:
sure, I hadn't looked at the others and just saw the root one